src/ga/ga_core.rs

```diff
@@ -27,14 +27,14 @@ impl Default for GAFlags
 pub trait GAIndividual
 {
     // Instance
-    fn crossover(&self, other : &Self) -> Box<Self>;
-    fn mutate(&mut self, pMutation : f32);
-    fn evaluate(&mut self) -> f32;
```

Comment:
I'm a bit torn with removing evaluate and moving it into the population, or leaving it as is. Comments? @carlos-lopez-garces @jissag